bin/startPipeline.sh

```diff
@@ -202,13 +202,13 @@ function submitPipeline () {
 			columnName="${sampleSheetColumnNames[${offset}]}"
 		fi
 		sampleSheetColumnOffsets["${columnName}"]="${offset}"
-		log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "${columnName} and sampleSheetColumnOffsets["${columnName}"] offset ${offset} "
+		log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "${columnName} and sampleSheetColumnOffsets [${columnName}] offset ${offset} "
 	done
 	prio="false"
-	if [[ ! -z "${sampleSheetColumnOffsets['FirstPriority']+isset}" ]]
+	if [[ -n "${sampleSheetColumnOffsets['FirstPriority']+isset}" ]]
 	then
 		priorityFieldIndex=$((${sampleSheetColumnOffsets['FirstPriority']} + 1))
-		priority="$(tail -n +2 ${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT} | awk -v prio=${priorityFieldIndex} 'BEGIN {FS=","}{print $prio}')"
+		priority=$(tail -n +2 "${TMP_ROOT_DIR}/projects/${_project}/${_run}/jobs/${project}.${SAMPLESHEET_EXT}" | awk -v prio="${priorityFieldIndex}" 'BEGIN {FS=","}{print "$prio"}')
```

Comment:
Deze moet je wel even goed testen, want de "$prio" is geen Bash variabele, maar een awk variabele en ik weet niet of je er nu geen "" bij geprint krijgt door awk...

Reply:
Hetzelfde construct met awk zit op nog wel meer plaatsen...